src/main/java/com/ryszardpanda/Library/repository/BookRepository.java

```diff
@@ -0,0 +1,95 @@
+package com.ryszardpanda.Library.repository;
+
+import com.ryszardpanda.Library.model.BookStats;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+import com.ryszardpanda.Library.model.Book;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Repository
+@RequiredArgsConstructor
+public class BookRepository {
+
+    private final List<Book> books;
+
+    public Book addBook(Book book) {
+        book.setId(books.size() + 1);
```

Comment:
book.setId(books.size() + 1); <- błąd w sytuacji kiedy dodałbym ksiązkę 1 2 i 3 
        to mialbym size = 3 
        usunalbym ksiązkę o id = 2 
        to mialbym size = 2 i w liscie ksiazki o id = 1 , id = 3.
        
        czyli dodanie kolejnej książki spowoduje ze zostanie nadany id = 3 <- błąd bo mamy wtedy dwie ksiazki o takim samym id.
        
        Znajdz max id w liscie i dodaj do tego +1

Reply:
done